src/ADDING_NEW_DATASETS.md

```diff
@@ -96,53 +94,53 @@ ecb_violations:
   schema:
     table_name: ecb_violations
     fields:
-      isn_dob_bis_extract: text
-      ecb_violation_number: text
-      ecb_violation_number: text
-      dob_violation_number: text
+      IsnDobBisExtract: text
```

Comment:
I said it was essential to have `PascalCased` and I included `snake_cased` in the guide. ha